src/tyr/route_serializer_osrm.cc

```diff
@@ -180,6 +184,38 @@ std::unordered_map<std::string, std::string> iso2_to_iso3 =
      {"VI", "VIR"}, {"VN", "VNM"}, {"VU", "VUT"}, {"WF", "WLF"}, {"WS", "WSM"}, {"YE", "YEM"},
      {"YT", "MYT"}, {"ZA", "ZAF"}, {"ZM", "ZMB"}, {"ZW", "ZWE"}, {"CS", "SCG"}, {"AN", "ANT"}};
 
+// Sign style and unit conventions for speed limit signs by country.
+// Most countries use Vienna style and km/h, but the countries below
+// use MUTCD and/or mph conventions.
+std::unordered_map<std::string, std::pair<std::string, std::string>> speed_limit_info = {
+    {"AG", {kSpeedLimitSignVienna, kSpeedLimitUnitsMph}},
```

Comment:
`AI` (Anguilla) also [uses](https://en.wikipedia.org/wiki/Road_signs_in_the_United_Kingdom#Virgin_Islands) mph with the Vienna sign.